examples/gatsby/gatsby-ssg/src/pages/ExperienceSlugExample.tsx

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import { fetchById, detachExperienceStyles } from '@contentful/experiences-sdk-react';
+import { useContentfulClient } from '../utils/useContentfulClient';
+
+exports.createPages = async ({ graphql, actions, reporter }: any) => {
```

Comment:
I get some errors when trying to run `npm run build`:

```
ERROR #11328  API.NODE.VALIDATION

The plugin "gatsby-plugin-page-creator" created a page without a valid default export.
```

This should create a static site with all those experiences prebuilt. 

Ideally, this would create the experiences to follow the same routing structure as the other apps (www.mysite.com/{slug}) and create a page for each experience that is in the space being used.